packages/core/src/i18n/translations/en.js

```diff
@@ -31,6 +31,7 @@ export default {
   'date-hint': 'Please enter 2 digits for the month and day and 4 digits for the year',
   'date-hint-with-select': 'For example: January 19 2000',
   'date-error': 'Please enter a complete date',
+  'number-error': 'Please enter a valid number',
```

Comment:
Can we add the new translation string(s) to our translations spreadsheet? https://docs.google.com/spreadsheets/d/1iIg9pP5GG-QFk7DrsyoKJv9GebtYmDuzKf_heqxWks4/edit#gid=0

I'm also not completely sure if the Spanish and Tagalog are correct.

Reply:
I added that to the spreadsheet, I don't really know what all the fields mean so please double check my work. I used google translate for the Spanish and Tagalog translations, so they are probably wrong. Is there a process for translations to follow/translators to verify strings?